src/sections/changableBlock/ChangableBlock.jsx

```diff
@@ -0,0 +1,30 @@
+import './ChangableBlock.css'
+import { Portfolio } from '../portfolio'
+import { Info } from '../info'
+import { useState } from 'react'
+
+export const ChangableBlock = () => {
+    const [block, setBlock] = useState(<Info/>);
```

Comment:
don't code like this
add flag
`const [showPortfolio, setShowPortfolio] = useState(false);`